src/getSequenceReducer.js

```diff
@@ -17,13 +16,26 @@ type SubReducers = {
   FINALLY ?:() => any;
 };
 
-type SequenceReducer = (state :any, action :SequenceAction, subReducers :SubReducers) => any;
+type ActionOrActionType =
+  | string
```

Comment:
We've always followed this pattern...

```
return getAllThings.reducer(state, action, {
  REQUEST: () => { ... },
  SUCCESS: () => { ... },
  FAILURE: () => { ... },
  FINALLY: () => { ... },
});
```

My change is meant to be backwards compatible with that, but I'd like to move towards this pattern:

```
return getAllThings.reducer(state, action.type, {
  REQUEST: () => { ... },
  SUCCESS: () => { ... },
  FAILURE: () => { ... },
  FINALLY: () => { ... },
});
```

Simply, instead of passing the entire `action` object, we'd pass only `action.type` because that's the only thing that matters.